src/centos/7/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN yum install -y \
         gcc \
         gcc-c++ \
         git \
+        gdb \
```

Comment:
The Docker convention is to alphabetize the packages.  It makes for better readability.  This applies throughout the PR.